examples/lines_bars_and_markers/marker_fillstyle_reference.py

```diff
@@ -9,30 +9,25 @@
 :doc:`/gallery/lines_bars_and_markers/marker_fillstyle_reference`
 and :doc:`/gallery/shapes_and_collections/marker_path` examples.
 """
+
 import numpy as np
 import matplotlib.pyplot as plt
 from matplotlib.lines import Line2D
 
 
 points = np.ones(5)  # Draw 3 points for each line
```

Comment:
It does draw *5* points and not 3 as the comment states, does it not?

Reply:
... sure